lib/fastlane/plugin/aws_device_farm/actions/aws_device_farm_action.rb

```diff
@@ -224,6 +223,14 @@ def self.available_options
             is_string:   true,
             optional:    false
           ),
+          FastlaneCore::ConfigItem.new(
+            key:         :network_profile_arn,
+            env_name:    'FL_AWS_DEVICE_FARM_NETWORK_PROFILE_ARN',
+            description: 'Network profile arn you want to use for running the applications',
+            default_value: 'arn:aws:devicefarm:us-west-2::networkprofile:public4',
```

Comment:
before that PR - the `network_profile_arn` was `nil`.
to be BC please set the default value to nil.

and later on something like

```ruby
if params[:network_profile_arn]
  configuration[:network_profile_arn] = params[:network_profile_arn]
end
```

this way its ensured that there is no change for current users.

Reply:
okay that makes sense to me, I made the value as optional and if there is an existing param passed it will use that else default value will be nil.